starky/src/verifier.rs

```diff
@@ -181,24 +196,24 @@ where
     ensure!(next_values.len() == S::COLUMNS);
     ensure!(quotient_polys.len() == stark.num_quotient_polys(config));
 
-    if stark.uses_permutation_args() {
-        let permutation_zs_cap = permutation_zs_cap
+    if stark.uses_lookups() {
+        let auxiliary_polys_cap = auxiliary_polys_cap
             .as_ref()
-            .ok_or_else(|| anyhow!("Missing Zs cap"))?;
-        let permutation_zs = permutation_zs
+            .ok_or_else(|| anyhow!("Missing auxiliary_polys_cap"))?;
+        let auxiliary_polys = auxiliary_polys
             .as_ref()
-            .ok_or_else(|| anyhow!("Missing permutation_zs"))?;
-        let permutation_zs_next = permutation_zs_next
+            .ok_or_else(|| anyhow!("Missing auxiliary_polys"))?;
+        let auxiliary_polys_next = auxiliary_polys_next
             .as_ref()
-            .ok_or_else(|| anyhow!("Missing permutation_zs_next"))?;
+            .ok_or_else(|| anyhow!("Missing auxiliary_polys_next"))?;
 
-        ensure!(permutation_zs_cap.height() == cap_height);
-        ensure!(permutation_zs.len() == num_zs);
-        ensure!(permutation_zs_next.len() == num_zs);
+        ensure!(auxiliary_polys_cap.height() == cap_height);
+        ensure!(auxiliary_polys.len() == num_auxiliary);
+        ensure!(auxiliary_polys_next.len() == num_auxiliary);
     } else {
-        ensure!(permutation_zs_cap.is_none());
-        ensure!(permutation_zs.is_none());
-        ensure!(permutation_zs_next.is_none());
+        ensure!(auxiliary_polys_cap.is_none());
```

Comment:
Nit: Aren't these checks (the `else` part) already ensured in `check_lookup_options`?